notify/discord/discord.go

```diff
@@ -34,10 +36,26 @@ import (
 )
 
 const (
-	// https://discord.com/developers/docs/resources/channel#embed-object-embed-limits - 256 characters or runes.
+	// https://discord.com/developers/docs/resources/channel#create-message
+	// 2000 characters per message is the limit (not counting embeds).
+	maxMessageContentLength = 2000
+	// 10 embeds per Message is the maximum.
+	maxEmbedsPerMessage = 10
```

Comment:
A concern I have here is that it's not uncommon for users to have large groups (more than 10 alerts). If Discord is limited to 10 embeds then this might make notifications less useful for those users as truncation will become a regular occurrence for them. What do you think @gotjosh?

Reply:
🤔  good point, I think this is OK for now - I really like the look of this notification as a whole. If users find 10 is too minimal for them to pay attention (in single group) we can always improve it later on. 

All I care is that when we have more than 10, we provide a clear message that states we couldn't fit them all.